core/src/main/java/nl/nn/adapterframework/receivers/JavaListener.java

```diff
@@ -143,7 +143,7 @@ private Message processRequest(String correlationId, M rawMessage, Message messa
 		}
 		log.debug("JavaListener [{}] processing correlationId [{}]" , getName(), correlationId);
 		if (context != null) {
-			Object object = context.get("httpRequest");
+			Object object = context.get("httpRequest"); //TODO dit moet weg
```

Comment:
Waar moet het wel naar toe eigenlijk?

Reply:
De prullenbak :D Het zit alleen in de streampipe, een nn specifiek ding...